stable/admin/templates/ingress.yaml

```diff
@@ -0,0 +1,83 @@
+{{- if .Values.admin.ingress }}
+{{- if eq (include "defaultfalse" .Values.admin.ingress.enabled) "true" }}
+{{- if .Values.admin.ingress.api }}
+{{- if .Values.admin.ingress.api.hostname }}
+apiVersion: networking.k8s.io/v1
```

Comment:
We need to document that Ingress can be enabled only with Kubernetes v1.19+.
I didn't include the `networking.k8s.io/v1beta1` version of the Ingress resource.